osu.Game/Beatmaps/Drawables/Cards/BeatmapCardContentBackground.cs

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable enable
+
+using System;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Colour;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Game.Overlays;
+
+namespace osu.Game.Beatmaps.Drawables.Cards
+{
+    public class BeatmapCardContentBackground : ModelBackedDrawable<IBeatmapSetOnlineInfo>
+    {
+        public IBeatmapSetOnlineInfo BeatmapSet
+        {
+            get => Model;
+            set => Model = value;
+        }
+
+        public new bool Masking
+        {
+            get => base.Masking;
+            set => base.Masking = value;
+        }
+
+        public BindableBool Dimmed { get; private set; } = new BindableBool();
```

Comment:
```suggestion
        public readonly BindableBool Dimmed = new BindableBool();
```

Reads weird to me having a `private set` when the only time it's set is right here.